cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -117,12 +119,14 @@ def registrationWorkflow(self, current_user):
         except ClientError as e:
             self.logger.error(e.response['Error']['Message'])
 
-    def addVisitEntry(self, current_user, location):
+    def addVisitEntry(self, current_user, location, tool):
+        
         # Get the current date at which the user logs in.
-        visit_date = datetime.datetime.now().timestamp()
+        eastern_tz = dateutil.tz.gettz('US/Eastern')
+        visit_date = datetime.datetime.now(tz=eastern_tz)
+        
         # Convert visit_date to human-readable format
-        visit_date = datetime.datetime.fromtimestamp(
-            visit_date).strftime('%Y-%m-%d %H:%M:%S')
+        visit_date = visit_date.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
I've gotta push back on storing time in our database in a human-readable format. The unix timestamp is much easier for computers to store and manipulate than a human-readable format.

It is better to store things in your database in a way that makes sense for your application, and then convert them to human-readable at presentation time.

Reply:
Makes sense ill go ahead and make that change aswell